ark/utils/segmentation_utils.py

```diff
@@ -176,3 +176,55 @@ def concatenate_csv(base_dir, csv_files, column_name="point", column_values=None
             combined_data = pd.concat((combined_data, temp_data), axis=0, ignore_index=True)
 
     combined_data.to_csv(os.path.join(base_dir, "combined_data.csv"), index=False)
+
+
+def visualize_segmentation(segmentation_labels_xr, channel_data_xr,
+                           output_dir,
+                           overlay_channels, fovs=None,
+                           save_tifs='overlays'):
+    """Runs the watershed transform over a set of probability masks output by deepcell network
+    Saves xarray to output directory
+
+    Args:
+        segmentation_labels_xr (xarray.DataArray): xarray containing segmentation labels
+        channel_data_xr (xarray.DataArray): xarray containing TIFs
+        output_dir (str): path to directory where the output will be saved
+        overlay_channels (tuple): channels to overlay segmentation output over
+        fovs (numpy.ndarray): field of view
+        save_tifs (str): flag to control what level of output to save.
+            Must be one of:
+
+            * 'all'
+            * 'none'
+            * 'overlays' (saves color overlays and segmentation masks)
+    """
+
+    if fovs is None:
+        fovs = segmentation_labels_xr.fovs
+    for fov in fovs:
+        if save_tifs == 'none':
```

Comment:
I think we can delete save_tifs completely. There's not really a scenario where people are going to want to visualize just one image in the notebook at a time. It's better to have them get created as files. 

You should make chan_list an optional parameter. If it's supplied, create those overlay tifs. If not, just create the segmentation_labels and segmentation_borders files